nexus/src/db/datastore/vpc.rs

```diff
@@ -675,4 +705,66 @@ impl DataStore {
                 )
             })
     }
+
+    /// Identify all subnets in use by each VpcSubnet
+    pub async fn resolve_subnets_to_ips<T: IntoIterator<Item = Name>>(
```

Comment:
This function name and / or comment could be a bit clearer. Maybe `resolve_vpc_subnets_to_ip_networks`? Since the word "subnet" is overloaded, I've generally found it useful to be really pedantic when they're both in play.

Reply:
Good suggestion, done in 8ab92afb.